src/App.js

```diff
@@ -1,21 +1,11 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <a
-          className="App-link"
-          href="https://github.com/ishimwepati"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          WazaCode Student at Microverse : click here to visit my Github.
-        </a>
-        <h1>Hello World</h1>
-      </header>
+      <Calculator />
```

Comment:
- [ ] I am putting this comment here because this is the best place I can put it since some of your files are not changed in this PR, kindly remove all boilerplates created when creating this app, I marked some of them in the image below
![image](https://github.com/ishimwepati/math-magicians/assets/66097371/3c8c13a1-73ca-4465-b89a-975b310abb7a)